app/org/sagebionetworks/bridge/json/ScheduleDeserializer.java

```diff
@@ -20,7 +23,7 @@
 import com.fasterxml.jackson.databind.DeserializationContext;
 import com.fasterxml.jackson.databind.JsonDeserializer;
 import com.fasterxml.jackson.databind.JsonNode;
-import com.google.common.collect.Lists;
+import com.fasterxml.jackson.databind.node.ArrayNode;
 
 public class ScheduleDeserializer extends JsonDeserializer<Schedule> {
```

Comment:
It looks from the SDK like we can get rid of the schedule serializer/deserializer, although there we include the jackson-datatype-joda package. Testing this now.